src/util/Errors.js

```diff
@@ -42,12 +42,30 @@ define(['okta', './Enums'], function (Okta, Enums) {
   }
   AuthStopPollInitiationError.prototype = new Error();
 
+  function U2FError (err) {
+    this.name = Enums.U2F_ERROR;
+    this.message = err.xhr.responseJSON.errorSummary;
+    this.errorSummary = err.xhr.responseJSON.errorSummary;
```

Comment:
since both `message` and `errorSummary` are the same, do we need both? I guess in the future some errors might not have the same content, but not sure if we want to just duplicate the data.